### DISCUSSION TITLE
Problem with Patched Talkgroups

### DISCUSSION THREAD

**Discussion by Crockpot2436** [chuot/rdio-scanner#299] [General]
Hello all, so I am having a problem with patched talkgroups with Rdio-Scanner and SDRTrunk. So I have turned the Auto Populate function off but somehow it still creates new talkgroups on Rdio-Scanner with the patched TGID. Is there any other way to turn this off where it will not create any other TG's than what I have specified in the system settings? I have attached an image below to help identify what I am talking about. Thank you very much!

![IMG_4613](https://user-images.githubusercontent.com/120535787/209727897-29f14b05-cc29-4c1c-ba6b-39d456678cd2.PNG)

**Comment by chuot** [comment]
Make sure the auto populate option is disabled under config/options but also for your particular system under config/system/...

It should then not create or modify any labels.
